rollout.py

```diff
@@ -99,6 +104,15 @@ def report(info):
         print(f"latest entry {version} on {stream} is not a rollout")
         return
 
+    latest_info = "unvalidated"
+    if not skip_build_id_check:
+        releases_url = RELEASES.substitute(stream=stream)
+        releases = requests.get(releases_url).json()["releases"]
+        if releases[-1]["version"] == version:
+            latest_info = "latest"
+        else:
```

Comment:
As an extra feature, it'd be straightforward to distinguish between stale and bogus versions here.